GW2EIEvtcParser/EIData/CombatReplay/CombatReplay.cs

```diff
@@ -295,6 +295,23 @@ internal void AddTether(IReadOnlyList<AbstractBuffEvent> tethers, string color)
                 }
             }
         }
+
+        internal void AddTetherByEffectGUID(EffectEvent effect, string color, int duration = 0, bool overrideDuration = false)
+        {
+            (int, int) lifespan;
+            if (effect.Duration > 0 && overrideDuration == false)
+            {
+                lifespan = ((int)effect.Time, (int)effect.Time + (int)effect.Duration);
```

Comment:
Should use ProfHelper.ComputeEffectLifespan and you can remove effect.Duration > 0 as that test is done in that API